docs/API/configure-buffers.md

```diff
@@ -0,0 +1,115 @@
+# Configuration Guidelines
+
+## Receiver Buffer Size
+
+The receiver buffer can be configured with the `SRTO_RCVBUF` socket option.
+Buffer size in bytes is expected to be passed in the `optval` argument of the `srt_setsockopt(..)` function.
+However, internally the value will be converted into the number of packets stored in the receiver buffer.
+
+The allowed value of `SRTO_RCVBUF` is also limited by the value of the flow control window size `SRTO_FC` socket option.
+See issue [#700](https://github.com/Haivision/srt/issues/700).
+
+The default flow control window size is 25600 packets. It is approximately:
+
+- **270 Mbits** of payload in the default live streaming configuration with an SRT payload size of **1316 bytes**;
+- **300 Mbits** of payload in the default file transfer configuration with an SRT payload size of **1456 bytes**.
+
+The default receiver buffer size is 8192 packets. It is approximately: 
+- **86 Mbits** of payload with the effective SRT payload size of **1316 bytes**.
+
+### Setting Receiver Buffer Size
+
+As already mentioned, the maximum allowed size of the receiver buffer is limited by the value of `SRTO_FC`.
+When the`SRTO_RCVBUF` option value is set using the`srt_setsockopt(..)` the function internally converts the provided size in bytes to the corresponding size in packets
```

Comment:
1) I think the word "function" is missing right after "set using the`srt_setsockopt(..)`"
2) The space is missing after "the" here: "When the`SRTO_RCVBUF` option"
3) The space is missing after "the" here: "is set using the`srt_setsockopt(..)`"

When the `SRTO_RCVBUF` option value is set using the `srt_setsockopt(..)` function, it [`srt_setsockopt(..)` ] internally converts the provided size in bytes to the corresponding size in packets ...